### ISSUE TITLE
滚字的笔顺不对

### ISSUE THREAD

**Issue by mijimoji** [chanind/hanzi-writer#136]
滚字的笔顺，在  六这们位置，是先写了六字下面的捺，然后写撇。
百度了一下，正确的应该是先写六字下面的撇，然后写捺。

**Comment by chanind** [comment]
Good catch! That does appear to be incorrect. I opened an issue in makemeahanzi here https://github.com/skishore/makemeahanzi/issues/72 since that's where this data comes from.
